error: argument 2 of '__atomic_load' discards 'volatile' qualifier

# Issue Description
Build of v2.0.0 fails on openSUSE Tumbleweed on `__atomic_load`:
```
[   45s] FAILED: gst/nnstreamer/libnnstreamer.so.p/_home_abuild_rpmbuild_BUILD_nnstreamer-2.0.0_gst_nnstreamer_tensor_filter_tensor_filter_common.c.o 
[   45s] cc -Igst/nnstreamer/libnnstreamer.so.p -Igst/nnstreamer -I../gst/nnstreamer -Igst/nnstreamer/include -I../gst/nnstreamer/include -Igst/nnstreamer/tensor_transform -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/include/gstreamer-1.0 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/orc-0.4 -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -Werror -std=gnu89 '-DVERSION="2.0.0"' '-DVERSION_MAJOR="2"' '-DVERSION_MINOR="0"' '-DVERSION_MICRO="0"' -Wwrite-strings -Wformat -Wformat-nonliteral -Wformat-security -Winit-self -Waddress -Wno-multichar -Wvla -Wpointer-arith -Wmissing-declarations -Wmissing-include-dirs -Wmissing-prototypes -Wnested-externs -Waggregate-return -Wold-style-definition -Wdeclaration-after-statement '-DNNSTREAMER_CONF_FILE="/etc/nnstreamer.ini"' -DGLIB_USE_G_MEMDUP2 -DENABLE_PROTOBUF=1 -DENABLE_TENSORFLOW=1 -DENABLE_TENSORFLOW2_LITE=1 -DHAVE_ORC=1 -Wredundant-decls -fPIC -pthread -MD -MQ gst/nnstreamer/libnnstreamer.so.p/_home_abuild_rpmbuild_BUILD_nnstreamer-2.0.0_gst_nnstreamer_tensor_filter_tensor_filter_common.c.o -MF gst/nnstreamer/libnnstreamer.so.p/_home_abuild_rpmbuild_BUILD_nnstreamer-2.0.0_gst_nnstreamer_tensor_filter_tensor_filter_common.c.o.d -o gst/nnstreamer/libnnstreamer.so.p/_home_abuild_rpmbuild_BUILD_nnstreamer-2.0.0_gst_nnstreamer_tensor_filter_tensor_filter_common.c.o -c /home/abuild/rpmbuild/BUILD/nnstreamer-2.0.0/gst/nnstreamer/tensor_filter/tensor_filter_common.c
[   45s] In file included from /usr/include/glib-2.0/glib/gthread.h:32,
[   45s]                  from /usr/include/glib-2.0/glib/gasyncqueue.h:32,
[   45s]                  from /usr/include/glib-2.0/glib.h:32,
[   45s]                  from ../gst/nnstreamer/hw_accel.h:19,
[   45s]                  from /home/abuild/rpmbuild/BUILD/nnstreamer-2.0.0/gst/nnstreamer/tensor_filter/tensor_filter_common.c:28:
[   45s] /home/abuild/rpmbuild/BUILD/nnstreamer-2.0.0/gst/nnstreamer/tensor_filter/tensor_filter_common.c: In function 'accl_hw_get_type':
[   45s] /usr/include/glib-2.0/glib/gatomic.h:113:5: error: argument 2 of '__atomic_load' discards 'volatile' qualifier [-Werror=incompatible-pointer-types]
[   45s]   113 |     __atomic_load (gapg_temp_atomic, &gapg_temp_newval, __ATOMIC_SEQ_CST); \
[   45s]       |     ^~~~~~~~~~~~~
[   45s] /usr/include/glib-2.0/glib/gthread.h:260:7: note: in expansion of macro 'g_atomic_pointer_get'
[   45s]   260 |     (!g_atomic_pointer_get (location) &&                             \
[   45s]       |       ^~~~~~~~~~~~~~~~~~~~
[   45s] /home/abuild/rpmbuild/BUILD/nnstreamer-2.0.0/gst/nnstreamer/tensor_filter/tensor_filter_common.c:2767:7: note: in expansion of macro 'g_once_init_enter'
[   45s]  2767 |   if (g_once_init_enter (&g_accl_hw_type_id__volatile)) {
[   45s]       |       ^~~~~~~~~~~~~~~~~
[   45s] cc1: all warnings being treated as errors
```


Expected Result
============
Build should succeed.

How to Reproduce
===============
Try to build with GCC11.


Further Information
===============
* x86_64: https://build.opensuse.org/package/live_build_log/home:Guillaume_G:branches:science:machinelearning/nnstreamer/openSUSE_Tumbleweed/x86_64
* aarch64: https://build.opensuse.org/package/live_build_log/home:Guillaume_G:branches:science:machinelearning/nnstreamer/openSUSE_Tumbleweed/aarch64

--- a/gst/nnstreamer/tensor_filter/tensor_filter_common.c
+++ b/gst/nnstreamer/tensor_filter/tensor_filter_common.c
@@ -2762,9 +2762,9 @@ parse_accl_hw_fill (parse_accl_args accl_args)
 static GType
 accl_hw_get_type (void)
 {
-  static volatile gsize g_accl_hw_type_id__volatile = 0;
+  static gsize g_accl_hw_type_id_store = 0;
 
-  if (g_once_init_enter (&g_accl_hw_type_id__volatile)) {
+  if (g_once_init_enter (&g_accl_hw_type_id_store)) {
     static const GEnumValue values[] = {
       {ACCL_NONE, ACCL_NONE_STR, ACCL_NONE_STR},
       {ACCL_DEFAULT, ACCL_DEFAULT_STR, ACCL_DEFAULT_STR},
@@ -2788,10 +2788,10 @@ accl_hw_get_type (void)
 
     GType g_accl_hw_type_id =
         g_enum_register_static (g_intern_static_string ("accl_hw"), values);
-    g_once_init_leave (&g_accl_hw_type_id__volatile, g_accl_hw_type_id);
+    g_once_init_leave (&g_accl_hw_type_id_store, g_accl_hw_type_id);
   }
 
-  return g_accl_hw_type_id__volatile;
+  return g_accl_hw_type_id_store;
 }
 
 /**

--- a/tests/common/unittest_common.cc
+++ b/tests/common/unittest_common.cc
@@ -948,7 +948,7 @@ TEST (commonTensorsConfig, validateInvalidParam2_n)
  */
 TEST (commonTensorsConfig, fromStructreInvalidParam0_n)
 {
-  GstStructure structure;
+  GstStructure structure = { 0 };
 
   EXPECT_FALSE (gst_tensors_config_from_structure (NULL, &structure));
 }

